src/Components/Cards.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import "./Navbar.css"
+import { Link } from 'react-router-dom'
+
+
+const Card = (props) => {

```

Comment:
Instead of (props) try destructuring the props ({img, alt, title, description}) this way everytime you don't have to write props.__key__ .